packages/react-components/src/components/RichTextEditor/RichTextSendBoxUtils.ts

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// Add component type check to assist in identification for usePropsFor
+// to avoid issue where production build does not have the component name
+
+/* @conditional-compile-remove(rich-text-editor) */
+/**
+ * @internal
+ */
+export const _isRichTextSendBox = Symbol('isRichTextSendBox');
```

Comment:
nit: as this is the property name, perhaps rename to something like
`richTextSendBoxIdentifier`